modules/core/src/v2/coins/algo.ts

```diff
@@ -244,10 +239,22 @@ export class Algo extends BaseCoin {
       changeAmount: 0,
       fee,
       changeOutputs: [],
-      memo
+      memo,
     };
   }
 
+  isStellarSeed(seed: string): boolean {
+    return stellar.StrKey.isValidEd25519SecretSeed(seed);
+  }
+
+  convertFromStellarSeed(seed: string): string {
+    // assume this is a trust custodial seed if its a valid ed25519 prv
+    if (!this.isStellarSeed(seed)) {
+      return null;
```

Comment:
Why not just return the same one you receive in the params? This way, this method will always return a valid seed, and there is no risk of hitting NPE

Reply:
Good call, this should really throw an Error instead of returning null.

Reply:
https://bitgoinc.atlassian.net/browse/CT-630